config/clusters/m2lines/support.values.yaml

```diff
@@ -11,6 +11,17 @@ grafana:
       - secretName: grafana-tls
         hosts:
           - grafana.m2lines.2i2c.cloud
+  grafana.ini:
+    server:
+      root_url: https://grafana.m2lines.2i2c.cloud
+    auth.github:
+      enabled: true
+      allow_sign_up: true
```

Comment:
The docs said to set this to `false` but I noticed it was set to `true` for the `leap` hub. Also, if we use `false` we would probably need to ask the comm. rep. to set up something like a GitHub team so that not just _anyone_ can sign in and view the boards. This would add back-and-forth to the setup though, unless _we_ do this explicitly as a part of https://github.com/2i2c-org/infrastructure/issues/1806

Reply:
Tbf, this would ultimately be fixed if we instead authenticate against JupyterHub and allow hub admins to see the dashboards... Which could be a part of https://github.com/2i2c-org/infrastructure/issues/535? I'm not sure...